aspnetcore/fundamentals/host/platform-specific-configuration/samples/2.x/HostingStartupSample/HostingStartupApp/Pages/Index.cshtml.cs

```diff
@@ -16,12 +16,16 @@ public IndexModel(IConfiguration config)
         }
 
         public string[] LoadedHostingStartupAssemblies { get; private set; }
+        public string ServiceKey_Development { get; private set; }
+        public string ServiceKey_Production { get; private set; }
 
         public void OnGet()
         {
             LoadedHostingStartupAssemblies = 
```

Comment:
Why isn't all of this done in the constructor?

Reply:
Fine by me. I'll change it on the next commit.